internal/storage/testdata/valid_key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
- Use test certificate as it's done for "ext" and "expired"
- The validity of 1 year is not ok for test certificate